resources/home/dnanexus/utils/util_functions.py

```
@@ -118,8 +119,72 @@ def initialise_project() -> Tuple[str, str, str]:
+        x for x in select_tracks if x not in available_reference_tracks
+    ]
+    if invalid_tracks:
+        raise ValueError(
+            "Invalid track names provided to select from build URL"
+            f" tracks: {invalid_tracks}"
+        )
+
+    return [x for x in reference_tracks if x["name"].lower() in select_tracks]
+
+
+def check_session_track_order(session_tracks) -> list:
```

Ruff (0.8.0)

Missing type annotation for function argument `session_tracks`

(ANN001)